apps/server/cachedDefaults/testData/alternateTestPreferences.json

```diff
@@ -0,0 +1,63 @@
+{
+  "defaultProvider": "mx",
+
+  "supportedProviders": [
+    "sophtron"
+  ],
+
+  "defaultProviderVolume": {
```

Comment:
I think ideally these preferences would only have what they need to pass. We could get rid of all the institutionProviderVolumeMap, and all the references to mx